src/Find_Command.php

```diff
@@ -233,7 +234,7 @@ private function recurse_directory( $path ) {
 		// Don't recurse directories that probably don't have a WordPress installation.
 		if ( ! $this->skip_ignored_paths ) {
 			// Assume base path doesn't need comparison
-			$compared_path = preg_replace( '#^' . preg_quote( $this->base_path ) . '#', '', $path );
+			$compared_path = preg_replace( '#^' . preg_quote( $this->base_path, '/' ) . '#', '', $path );
```

Comment:
Uh.. oh... this is wrong... the delimiter should be passed as the second parameter and in this case, the delimiter is `#`.

Reply:
@schlessera, should I send a PR with `#` as the delimiter for `preg_quote`? I don't recall why I added `/` here.

Reply:
@jrfnl Ah, good catch, thanks for that!

@thrijith Yes, please.

Reply:
@thrijith Did you maybe intend to concatenate to add trailing slash here... ?

Reply:
Either way, `preg_quote()` should have a `#` as the second argument in this case.